lib/coverband/utils/source_file.rb

```diff
@@ -242,7 +242,7 @@ def gem?
       end
 
       def gem_name
-        gem? ? short_name.split('/').first.gsub(%r{^\.}, '') : nil
+        gem? ? short_name.split('/').first.gsub(/^\./, '') : nil
```

Comment:
this is fine, but I did disable this rubocop warning as the %r is better in some cases when there are slashes etc.